cmd/truss/main.go

```diff
@@ -29,6 +30,7 @@ var (
 	svcPackageFlag = flag.String("svcout", "", "Go package path where the generated Go service will be written. Trailing slash will create a NAME-service directory")
 	verboseFlag    = flag.BoolP("verbose", "v", false, "Verbose output")
 	helpFlag       = flag.BoolP("help", "h", false, "Print usage")
+	getStartedFlag = flag.BoolP("getstarted", "", false, "Output a 'getstarted.proto' protobuf file in ./")
```

Comment:
There is documentation, shown when you use the option. https://github.com/TuneLab/go-truss/pull/196/files#diff-b7a48309a1d9f912b7f8857bd593df6eR38

Because the `getStartedFlag` is actually a bool flag and the name you can provide is actually handled outside of the way the flag package canonically does assignment, I don't want to say in the help message that you can provide a name. If we did, someone might think it means you can pass `--getstarted=foo` which will not work and will cause the program to fail.

So instead, I put the longer, more explicit instructions on how to do it in the `nextSteps` message printed when you actually use `--getstarted`.